kogito-kafka-quickstart-quarkus/README.md

```diff
@@ -109,10 +166,22 @@ this will then trigger the successful processing of the traveller and put anothe
 }
 ```
 
+* To examine ProcessedTravellers topic and verify it was auto sent
+
+```
+kafka-console-consumer.sh --bootstrap-server localhost:9092 --topic processedtravellers --from-beginning
```

Comment:
As it is done for Kafka & Zookeeper starts, use `bin/kafka-console-consumer.sh` in kafka folder

Reply:
Done @radtriste